evergreen/run-tests.sh

```diff
@@ -21,18 +21,12 @@ TOPOLOGY=${TOPOLOGY:-server}
 
 provision_ssl () {
```

Comment:
I've refactored this function to take in a parameter that specifies which URI variable we should be modifying

Reply:
Acknowledged.